contributor_docs/method.example.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * "This is a template for inline documentation of a method. Remove all text
+ * between double quotes for using this template. Some description about the
+ * method goes here. Explain in simple words, what the function does and what
+ * would be good/bad use cases for it. If there are any corners cases or warnings,
+ * do explain them over here."
+ *
+ * By default, the background is transparent.
+ *
+ * @method "methodName"
+ * @param {dataType} "variableName" "Description of the parameter"
+ * @param {dataType} ["variableName"] "Description of optional parameter"
+ * @chainable
+ *
+ * @example
+ * <div>
+ * <code>
+ * // "Comment explaining the example."
+ * "Your code goes here."
+ * </code>
+ * </div>
+ *
+ * @example
```

Comment:
there should be only one example tag, so this line can be removed

Reply:
Ok